### PR TITLE
[SD-4495] - Macros for currency coversions

### PR DESCRIPTION
**Pull request by akintolga**
- Improved regex for matching the currencies
- Converting the currencies using the same precision of the originals

### CODE REVIEW

**Comment by marwoodandrew** [comment]
:+1:

File: superdesk/macros/currency_usd_to_aud.py [hunk 1]
```diff
@@ -26,11 +27,14 @@ def get_rate():
 def usd_to_aud(item, **kwargs):
     """Convert USD to AUD."""
 
-    rate = get_rate()
+    rate = kwargs.get('rate') or get_rate()
     if os.environ.get('BEHAVE_TESTING'):
         rate = USD_TO_AUD
 
-    return currency_base.do_conversion(item, rate, 'AUD %d', '\$([0-9]+)')
+    regex = r'([$USD]+|(USD)|(\$US))\s*\-?\s*\(?(((\d{1,3}((\,\d{3})*|\d*))?' \
```
Comment (mdhaman) [inline superdesk/macros/currency_usd_to_aud.py]: this will match DS 700.00 and any other combinations of `$USD`.

**Comment by mdhaman** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**